content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,526 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+You may have have different motivations for running a standalone kubelet.
+This tutorial is aimed at introducing you to Kubernetes, even if you don't have
+much experience with it. You can follow this tutorial and learn about node setup,
+basic (static) Pods, and how Kubernetes manages containers.
```

Comment:
Don't see value of this paragraph.

Reply:
Can you suggest a different one? The tutorial still needs an introduction. 
@sftim, we went through several iterations of this opening paragraph already.

Reply:
I drafted this, and I think it helps readers if we explain who our target audience are. I don't support removing it.

Reply:
My take is that this page is useful because it helps explain 1) how static Pods are managed by the "node agent"; 2) why there is a standalone mode for kubelet; 3) how to play with kubelet under standalone mode, i.e. insecure ports, no kubeconfig, no node registration, etc.

My concern is about this statement, "This tutorial is aimed at introducing you to Kubernetes, even if you don't have much experience with it. "